eng/native/functions.cmake

```diff
@@ -291,47 +291,52 @@ function(strip_symbols targetName outputFilename)
     endif (CLR_CMAKE_TARGET_OSX OR CLR_CMAKE_TARGET_IOS)
 
     set(${outputFilename} ${strip_destination_file} PARENT_SCOPE)
+  else(CLR_CMAKE_HOST_UNIX)
+    set(${outputFilename} ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/${targetName}.pdb PARENT_SCOPE)
   endif(CLR_CMAKE_HOST_UNIX)
 endfunction()
 
-function(install_symbols targetName destination_path)
-  strip_symbols(${targetName} strip_destination_file)
-
+function(install_symbols symbol_file destination_path)
   if(CLR_CMAKE_TARGET_WIN32)
-    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/${targetName}.pdb DESTINATION ${destination_path}/PDB)
+    install(FILES ${symbol_file} DESTINATION ${destination_path}/PDB)
```

Comment:
I think it is better, if we do not modify this (Windows) part at this stage, as it is working out fine. Otherwise we would need to update libraries and installer as well.

Reply:
I'll rather look into fixing the libraries and installer. I have forgotten that they use these now too.

Reply:
Sure, I remember doing similar cleanup in libraries, it was something like touching one file (`src/libraries/Native/Unix/CMakeLists.txt`) vs. multiple other CMakeLists.txt under libraries, therefore, ended up favoring the common pattern so to touch fewer files. :)